sim_learn.py

```diff
@@ -1,18 +1,15 @@
-from numpy import amin, amax, mean, array, append
-from numpy.random import RandomState
-from pypuf.learner.regression.logistic_regression import LogisticRegression
 from pypuf.simulation.arbiter_based.ltfarray import LTFArray
-from pypuf import tools
-import time
-from sys import argv, stdout, stderr
+from pypuf.experiments.experiment.logistic_regression import ExperimentLogisticRegression
+from pypuf.experiments.experimenter import Experimenter
+from sys import argv, stderr
 
 
 def main(args):
 
-    if len(args) != 10:
+    if len(args) < 10:
```

Comment:
additional arguments shouldn't be silently ignored

Reply:
Now it is ```len(args) < 10 or len(args) > 11``` https://github.com/nils-wisiol/pypuf/blob/e1d408389cff95a58dbaeb7cc39e702b99b6d4be/sim_learn.py#L9 .